src/UI/Components/Diagram/Canvas.test.tsx

```diff
@@ -277,7 +283,7 @@ beforeEach(() => {
   });
 });
 
-describe("Canvas.tsx", () => {
+describe.only("Canvas.tsx", () => {
```

Comment:
I think this is a left over from running your test locally :D

Reply:
right, removed :D